src/support_code_library_builder/build_helpers.js

```diff
@@ -51,7 +55,7 @@ export function buildTestRunHookDefinition({ options, code, cwd }) {
   })
 }
 
-export function buildStepDefinition({ pattern, options, code, cwd }) {
+export function buildStepDefinitionStruct({ pattern, options, code, cwd }) {
```

Comment:
Thoughts on naming this `buildStepDefinitionConfig`? Struct is too generic to me. The other method would become `buildStepDefinitionFromConfig`

Reply:
Done